modules/tinymce/src/themes/silver/main/ts/alien/NotificationManagerImpl.ts

```diff
@@ -23,14 +23,17 @@ export default (
 ): NotificationManagerImpl => {
   const sharedBackstage = extras.backstage.shared;
 
+  const getBoundsContainer = () => SugarElement.fromDom(editor.queryCommandValue('ToggleView') === '' ? editor.getContentAreaContainer() : editor.getContainer());
+
   const getBounds = () => {
-    const contentArea = Boxes.box(SugarElement.fromDom(editor.getContentAreaContainer()));
+    const contentArea = Boxes.box(getBoundsContainer());
     return Optional.some(contentArea);
   };
 
   const clampComponentsToBounds = (components: AlloyComponent[]) => {
     getBounds().each((bounds) => {
       Arr.each(components, (comp) => {
+        Css.set(comp.element, 'width', '');
```

Comment:
This was key to the problem, allowing the notification to expand before the constraint was applied so we didn't use the constrained value to measure subsequent updates 👍

I think we should remove rather than set empty though.
```suggestion
        Css.remove(comp.element, 'width');
```